keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -306,68 +345,34 @@ The Windows container runtime also has a few important differences:
+- `V1.Container.SecurityContext.seLinuxOptions` - not possible on Windows, no SELinux
+- `V1.Container.terminationMessagePath` - this has some limitations in that Windows doesn't support mapping single files. The default value is `/dev/termination-log`, which does work because it does not exist on Windows by default.
 
 
 ### V1.Pod
 
-> TODO: double check other fields in [source](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2743)
-
-
-`V1.Pod.hostIPC`, `v1.pod.hostpid` - host namespace sharing is not possible on Windows
-
-`V1.Pod.hostNetwork` - There is no Windows OS support to share the host network
-
-`V1.Pod.dnsPolicy` - ClusterFirstWithHostNet - is not supported because Host Networking is not supported on Windows.
-
-`V1.podSecurityContext.runAsUser` provides a UID, not available on Windows
-`V1.podSecurityContext.supplementalGroups` provides GID, not available on Windows
-
-`V1.Pod.shareProcessNamespace` - this is an alpha feature, and depends on Linux cgroups which are not implemented on Windows
-
-`V1.Pod.volumeDevices` - this is an alpha feature, and is not implemented on Windows. Windows cannot attach raw block devices to pods.
-
-`V1.Pod.Volumes` - EmptyDir, Secret, ConfigMap, HostPath - all work and have tests in TestGrid
+- `V1.Pod.hostIPC`, `v1.pod.hostpid` - host namespace sharing is not possible on Windows
+- `V1.Pod.hostNetwork` - There is no Windows OS support to share the host network
+- `V1.Pod.dnsPolicy` - ClusterFirstWithHostNet - is not supported because Host Networking is not supported on Windows.
+- `V1.podSecurityContext.runAsUser` provides a UID, not available on Windows
+- `V1.podSecurityContext.supplementalGroups` provides GID, not available on Windows
+- `V1.Pod.shareProcessNamespace` - this is an beta feature, and depends on Linux namespaces which are not implemented on Windows. Windows cannot share process namespaces or the container's root filesystem. Only the network can be shared.
+- `V1.Pod.terminationGracePeriodSeconds` - this is not fully implemented in Docker on Windows, see: [reference](https://github.com/moby/moby/issues/25982). The behavior today is that the ENTRYPOINT process is sent `CTRL_SHUTDOWN_EVENT`, then Windows waits 5 seconds by hardcoded default, and finally shuts down all processes using the normal Windows shutdown behavior. The 5 second default is actually in the Windows registry [inside the container](https://github.com/moby/moby/issues/25982#issuecomment-426441183), so it can be overridden when the container is built. Runtime configuration will be feasible in CRI-ContainerD but not for v1.14. Issue [#73434](https://github.com/kubernetes/kubernetes/issues/73434) is tracking this for a later release.
+- `V1.Pod.volumeDevices` - this is an beta feature, and is not implemented on Windows. Windows cannot attach raw block devices to pods.
+- `V1.Pod.Volumes` - EmptyDir, Secret, ConfigMap, HostPath - all work and have tests in TestGrid
```

Comment:
Nit: This is the Go field name whereas the above are json field names. But this can be cleaned up later. Definitely it should be consistent in user documentation. Ditto for the field below.